contracts/RewardEscrowV2.sol

```diff
@@ -379,7 +379,7 @@ contract RewardEscrowV2 is
                 unchecked {
                     amountToUnstake = totalWithFee - unstakedEscrow;
                 }
-                stakingRewards.unstakeEscrowSkipCooldown(msg.sender, amountToUnstake);
+                stakingRewards.unstakeEscrowAdmin(msg.sender, amountToUnstake);
```

Comment:
You have to call unstakeEscrowAdmin otherwise `msg.sender` for
```
function unstakeEscrow(uint256 _amount) external {
        _unstakeEscrow(msg.sender, _amount);
    }
```

would be the rewardEscrowV2 contract.